src/js/components/covid19/budgetCategories/BudgetCategories.jsx

```diff
@@ -143,16 +142,12 @@ const BudgetCategories = () => {
                 <Tabs active={activeTab} types={tabs} switchTab={changeActiveTab} />
             </div>
             <div className="overview-data-group">
-                {overviewData.map((data) => (
-                    <OverviewData
-                        key={data.label}
-                        {...data}
-                        isLoading={(
-                            data.type === 'count' &&
-                            inFlight
-                        )}
-                        amount={amounts[data.type]} />
-                ))}
+                <InformationBoxes
+                    boxes={overviewData.map((data) => ({
+                        ...data,
+                        isLoading: data.type === 'count' && inFlight,
```

Comment:
<img width="1056" alt="Screen Shot 2021-04-01 at 6 06 01 PM" src="https://user-images.githubusercontent.com/7108785/113359355-9ba7e500-9315-11eb-9063-5910578c3aaa.png">
Looks like this is an existing bug, but we may want to add a loading state for the other three data points (I think they are usually faster to load though)